### PR TITLE
Updated image utilities and package component.

### PR DESCRIPTION
**Pull request by ShreyashSaitwal**
## Types of changes:
- [x] Correct a spelling
- [x] Add new information
- [x] Change information

## Checklist:
- [x] I have proofread the changes. (Spellings, grammar, etc)
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: docs/components/utilities/package.md [hunk 1]
```diff
@@ -1,4 +1,139 @@
...
+![Is Permission Granted](/assets/images/components/user-interface/utilities/package/m_is-permission-granted.png)
+
+_**\(** Text `permission Name`, Text `package Name` **\)**_
+
+Checks whether a particular package (app) has been granted a particular permission.
+
+Params               |  []()       
+-------------------- | ------- 
+`permission Name`    | **Text:** The name of the permision that has to be checked. For example, READ_CALENDER.
+`package Name`       | **Text:**  The package name of the app whose name has to be returned. For example, io.kodular.myfirstapp
```
Comment (barreeeiroo) [inline docs/components/utilities/package.md]:
```suggestion
`package Name`       | **Text:**  The package name of the app whose name has to be returned. For example, _io.kodular.myfirstapp_.
```

File: docs/components/utilities/package.md [hunk 2]
```diff
@@ -1,4 +1,139 @@
...
+
+![Is Permission Granted](/assets/images/components/user-interface/utilities/package/m_is-permission-granted.png)
+
+_**\(** Text `permission Name`, Text `package Name` **\)**_
+
+Checks whether a particular package (app) has been granted a particular permission.
+
+Params               |  []()       
+-------------------- | ------- 
+`permission Name`    | **Text:** The name of the permision that has to be checked. For example, READ_CALENDER.
```
Comment (barreeeiroo) [inline docs/components/utilities/package.md]:
```suggestion
`permission Name`    | **Text:** The name of the permision that has to be checked. For example, _READ\_CALENDAR_.
```

File: docs/components/utilities/image-utilities.md [hunk 3]
```diff
@@ -1,4 +1,26 @@
...
+
+![Load Image Async](/assets/images/components/utilities/image-utilities/m_load-image-async.png)
+
+_**\(** Component `component`, Text `path` **\)**_
+
+Loads a new image from the given path to the specified component.
+
+Params               |  []()       
+-------------------- | ------- 
+`component`          | **Component:**  The component where the image has to be shown. This is set using the Component type block.(Found a                          the last of all the blocks of a component. )
```
Comment (barreeeiroo) [inline docs/components/utilities/image-utilities.md]:
```suggestion
`component`          | **Component:**  The desired component where the image has to be shown.
```

File: docs/components/utilities/package.md [hunk 4]
```diff
@@ -1,4 +1,139 @@
...
+
+![App Name From](/assets/images/components/user-interface/utilities/package/m_app-name-from.png)
+
+_**\(** Text `package Name` **\)**_
+
+Returns the name of the app from the given package name.
+
+Params               |  []()       
+-------------------- | ------- 
+`package Name`       | **Text:**  The package name of the app whose name has to be returned. For example, io.kodular.myfirstapp
```
Comment (barreeeiroo) [inline docs/components/utilities/package.md]:
```suggestion
`package Name`       | **Text:**  The package name of the app whose name has to be returned. For example, _io.kodular.myfirstapp_.
```

File: docs/components/utilities/package.md [hunk 5]
```diff
@@ -1,4 +1,139 @@
...
+
+![Get Package Icon](/assets/images/components/user-interface/utilities/package/m_get-package-icon.png)
+
+_**\(** Text `package Name` **\)**_
+
+Returns the icon of the app from the given package name.
+
+Params               |  []()       
+-------------------- | ------- 
+`package Name`       | **Text:**  The package name of the app whose name has to be returned. For example, io.kodular.myfirstapp
```
Comment (barreeeiroo) [inline docs/components/utilities/package.md]:
```suggestion
`package Name`       | **Text:**  The package name of the app whose name has to be returned. For example, _io.kodular.myfirstapp_.
```

File: docs/components/utilities/package.md [hunk 6]
```diff
@@ -1,4 +1,139 @@
...
+
+![Is Package Installed](/assets/images/components/user-interface/utilities/package/m_is-package-installed.png)
+
+_**\(** Text `package Name` **\)**_
+
+Returns true if the package (app) is installed.
+
+Params               |  []()       
+-------------------- | ------- 
+`package Name`       | **Text:**  The package name of the app whose name has to be returned. For example, io.kodular.myfirstapp
```
Comment (barreeeiroo) [inline docs/components/utilities/package.md]:
```suggestion
`package Name`       | **Text:**  The package name of the app whose name has to be returned. For example, _io.kodular.myfirstapp_.
```

File: docs/components/utilities/package.md [hunk 7]
```diff
@@ -1,4 +1,139 @@
...
+
+![Version Code From](/assets/images/components/user-interface/utilities/package/m_version-code-from.png)
+
+_**\(** Text `package Name` **\)**_
+
+Returns the version code of the app from the given package name.
+
+Params               |  []()       
+-------------------- | ------- 
+`package Name`       | **Text:**  The package name of the app whose name has to be returned. For example, io.kodular.myfirstapp
```
Comment (barreeeiroo) [inline docs/components/utilities/package.md]:
```suggestion
`package Name`       | **Text:**  The package name of the app whose name has to be returned. For example, _io.kodular.myfirstapp_.
```

File: docs/components/utilities/package.md [hunk 8]
```diff
@@ -1,4 +1,139 @@
...
+
+![Version Name From](/assets/images/components/user-interface/utilities/package/m_version-name-from.png)
+
+_**\(** Text `package Name` **\)**_
+
+Returns the version name of the app from the given package name.
+
+Params               |  []()       
+-------------------- | ------- 
+`package Name`       | **Text:**  The package name of the app whose name has to be returned. For example, io.kodular.myfirstapp
```
Comment (barreeeiroo) [inline docs/components/utilities/package.md]:
```suggestion
`package Name`       | **Text:**  The package name of the app whose name has to be returned. For example, _io.kodular.myfirstapp_.
```

File: docs/components/utilities/package.md [hunk 9]
```diff
@@ -1,4 +1,139 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (barreeeiroo) [inline docs/components/utilities/package.md]: Why so many blank lines at the bottom???

File: docs/components/utilities/package.md (hunk 1 above)
Comment (ShreyashSaitwal) [reply docs/components/utilities/package.md]: Ah! This branch has some problems. Closing this pull request for now, will open a new one.

**Pull request closed without merging**